.github/workflows/run-launchtimes-metrics.yaml

```diff
@@ -12,6 +12,11 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: get-kata-tarball
+        uses: actions/download-artifact@v3
+        with:
+          name: kata-static-tarball-amd64${{ inputs.tarball-suffix }}
```

Comment:
Gaby, in order to use `inputs.tarball-suffix`, you'd need to actually start receiving it as an input.

See: https://github.com/kata-containers/kata-containers/blob/decbe77e28a27a40bd02a8d5ef9a1b51b7ec9c21/.github/workflows/ci-on-push.yaml#L17
and https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/publish-kata-deploy-payload-amd64.yaml#L3-L7 for reference.

Reply:
@fidencio thanks I used it as an input

Reply:
@GabyCT basically you need to pass the tarball string from `ci-on-push` for example:

```
run-metrics-tests:
    needs: build-kata-static-tarball-amd64
    uses: ./.github/workflows/run-metrics.yaml
    with:
      tarball-suffix: -${{ github.event.pull_request.number}}-${{ github.event.pull_request.head.sha }}
```